CODEOWNERS

```diff
@@ -0,0 +1,146 @@
+/Platform/AMD/AgesaPkg/**       @changab @abdattar @pbgrimes
+/Platform/AMD/AmdCbsPkg/**      @changab @abdattar @pbgrimes
+/Platform/AMD/AmdCpmPkg/**      @changab @abdattar @pbgrimes
+
+# AMD Seattle platforms
+/Platform/AMD/OverdriveBoard/** @changab @abdattar @pbgrimes @ardbiesheuvel @leiflindholm
+/Silicon/AMD/Styx/**            @changab @abdattar @pbgrimes @ardbiesheuvel @leiflindholm
+/Platform/LeMaker/CelloBoard/** @ardbiesheuvel @leiflindholm
+/Platform/SoftIron/             @ardbiesheuvel @leiflindholm
+
+# AMD VanGoghBoard
+/Platform/AMD/VanGoghBoard/** @changab @abdattar @pbgrimes @exinghr
+
+# Ampere Computing
+/Platform/Ampere/** @nhivp
+/Silicon/Ampere/** @nhivp
+
+# ARM
+/Platform/ARM/** @samimujawar
+/Silicon/ARM/** @samimujawar
+
+# BeagleBoard:
+/Platform/BeagleBoard/** @ardbiesheuvel @leiflindholm
+/Silicon/TexasInstruments/** @ardbiesheuvel @leiflindholm
+
+# DisplayLink
+/Drivers/DisplayLink/** @leiflindholm @ardbiesheuvel
+
+/Features/Ext4Pkg/** @heatd
+
+# HiSilicon
+/Platform/Hisilicon/** @leiflindholm
+/Silicon/Hisilicon/** @leiflindholm
+
+/Features/Intel/** @nate-desimone @SaiChaganty
+/Features/Intel/Debugging/** @ydong10 @nate-desimone @SaiChaganty
+/Features/Intel/OutOfBandManagement/IpmiFeaturePkg/** @nate-desimone @SaiChaganty
+/Features/Intel/UserInterface/** @dandanbi @nate-desimone @SaiChaganty
+
+/Features/ManageabilityPkg/** @changab
+
+/Platform/Intel/** @nate-desimone @SaiChaganty
+/Platform/Intel/QuarkPlatformPkg/** @mdkinney @nate-desimone @SaiChaganty
+/Platform/Intel/Vlv2TbltDevicePkg/** @zailiangsun @yqian4 @nate-desimone @SaiChaganty
+/Platform/Intel/BoardModulePkg/** @ydong10 @nate-desimone @SaiChaganty
+/Platform/Intel/KabylakeOpenBoardPkg/** @ChaselChiu @nate-desimone @SaiChaganty
+/Platform/Intel/KabylakeOpenBoardPkg/AspireVn7Dash572G/** @benjamindoron @nate-desimone @SaiChaganty
+/Platform/Intel/KabylakeOpenBoardPkg/GalagoPro3/** @jackpot51
+/Platform/Intel/MinPlatformPkg/** @ChaselChiu @nate-desimone @SaiChaganty
+/Platform/Intel/PurleyOpenBoardPkg/** @nate-desimone @ChaselChiu @SaiChaganty
+/Platform/Intel/WhiskeylakeOpenBoardPkg/** @nate-desimone @SaiChaganty @ChaselChiu
+/Platform/Intel/CometlakeOpenBoardPkg/** @ChaselChiu @nate-desimone @SaiChaganty
+/Platform/Intel/TigerlakeOpenBoardPkg/** @SaiChaganty @nate-desimone
+/Platform/Intel/AlderlakeOpenBoardPkg/** @nate-desimone @SaiChaganty
+/Platform/Intel/WhitleyOpenBoardPkg/** @ChaselChiu  @nate-desimone @SaiChaganty
+/Platform/Intel/SimicsOpenBoardPkg/** @nate-desimone @SaiChaganty
+/Platform/Intel/Tools/** @BobCF @lgao4 @nate-desimone @SaiChaganty
+
+/Silicon/Intel/** @nate-desimone @SaiChaganty
+/Silicon/Intel/IntelSiliconPkg/** @niruiyu @nate-desimone @SaiChaganty
+/Silicon/Intel/QuarkSocPkg/** @mdkinney @nate-desimone @SaiChaganty
+/Silicon/Intel/Vlv2DeviceRefCodePkg/** @zailiangsun @yqian4 @nate-desimone @SaiChaganty
+/Silicon/Intel/CoffeelakeSiliconPkg/** @ChaselChiu @SaiChaganty @nate-desimone

```

Comment:
For Intel part "@nate-desimone @SaiChaganty" is nearly everywhere so would be nice to have them always in same order.